### PR TITLE
Added links to skills section and enhanced them

### LINKED ISSUE #427: Add links to topics given under SHARPEN YOUR SKILLS.
## Proposal

The topics which are specified under SHARPEN YOUR SKILLS section redirect to the homepage. Here, I would like to add the links to the topics which will redirect the user to the new webpage consisting the stuffs of the same. So, I would be creating webpages of each topic and add the link for it.
For ex. The 'DATA STRUCTURES' skill given under SHARPEN YOUR SKILLS section will redirect the user to a new webpage which will consist of all the stuffs regarding the same.

### PR DESCRIPTION
**Pull request by codebuzzer01**
# Description

Links to the specific site of that particular topic is added which lets user redirect to a new page for each skill and logo is enhanced as well.

# Fixes #427

## List any dependencies that are required for this change  
No

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
Screenshot (217)
Screenshot (218)
   - After :
Screenshot (219)
Required skill gets open in new tab:
Screenshot (220)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by codebuzzer01** [comment]
@ravikr126  I have resolved the above issue. Kindly review it.

**Review by ravikr126 — APPROVED** [review]

**Comment by codebuzzer01** [comment]
@harikesh409  I have resolved the above issue. Kindly review it.

File: index.html [hunk 1]
```diff
@@ -195,19 +197,19 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
-        <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="#">Java</a></th>
+         <th><i class="fab fa-cuttlefish"></i>&nbsp; <a href="https://www.youtube.com/watch?v=KJgsSFOSQv0" target="_blank">C</a></th>
+        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.cplusplus.com/" target="_blank">C++</a></th>
+        <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=eTXd89t8ngI&list=PLd3UqWTnYXOmx_J1774ukG_rvrpyWczm0" target="_blank">Java</a></th>
       </tr>
       <tr>
-        <th><i class="fas fa-gem"></i>&nbsp; <a href="#">Ruby</a></th>
-        <th><i class="fas fa-square-root-alt"></i>&nbsp; <a href="#">Mathematics</a></th>
-        <th><i class="fa fa-database"></i>&nbsp; <a href="#">Databases</a></th>
+        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=g-1Cn3ccwXY" target="_blank">STL</a></th>
```
Comment (harikesh409) [inline index.html:210]: STL is a part of CPP only so revert it back to Ruby only if you are unable to find any correct resources you can ask us or leave it blank.

File: index.html [hunk 2]
```diff
@@ -195,19 +197,19 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+        <th><i class="devicon-devicon-plain"></i>&nbsp; <a href="https://clist.by/" target="_blank">Contests</a></th>
```
Comment (harikesh409) [inline index.html:211]: Contents is not a skill so revert back to mathematics only.

File: index.html [hunk 3]
```diff
@@ -195,19 +197,19 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+        <th><i class="devicon-redhat-plain"></i>&nbsp; <a href="https://devpost.com/hackathons" target="_blank">Hackathons</a></th>
```
Comment (harikesh409) [inline index.html:212]: Hackathons is not a skill you can change to Databases or SQL.

File: index.html [hunk 4]
```diff
@@ -27,6 +27,8 @@
     <link rel="apple-touch-icon" sizes="180x180" href="favicon_package\apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="favicon_package\favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="favicon_package\favicon-16x16.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.11.0/devicon.min.css">
```
Comment (harikesh409) [inline index.html:30]: Don't include additional icon packs kindly remove them and use the existing packs only.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by codebuzzer01** [comment]
Ok @harikesh409  I will do the requested changes.

**Comment by codebuzzer01** [comment]
@harikesh409 

Screenshot (223)

**Comment by codebuzzer01** [comment]
@harikesh409  tell me if it's okay?

File: index.html [hunk 5]
```diff
@@ -207,9 +208,9 @@ <h1 class="text-center">Sharpen Your Skills</h1>
         <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=eTXd89t8ngI&list=PLd3UqWTnYXOmx_J1774ukG_rvrpyWczm0" target="_blank">Java</a></th>
       </tr>
       <tr>
-        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=g-1Cn3ccwXY" target="_blank">STL</a></th>
-        <th><i class="devicon-devicon-plain"></i>&nbsp; <a href="https://clist.by/" target="_blank">Contests</a></th>
-        <th><i class="devicon-redhat-plain"></i>&nbsp; <a href="https://devpost.com/hackathons" target="_blank">Hackathons</a></th>
+        <th><i class="devicon-ruby-plain"></i>&nbsp; <a href="https://www.youtube.com/watch?v=t_ispmWmdjY" target="_blank">Ruby</a></th>
```
Comment (harikesh409) [inline index.html:211]: Font awesome gem icon can be used here.

File: index.html [hunk 6]
```diff
@@ -207,9 +208,9 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+        <th><i class="devicon-go-plain"></i>&nbsp; <a href="https://www.youtube.com/watch?v=YS4e4q9oBaU" target="_blank">Golang</a></th>
+        <th><i class="devicon-swift-plain"></i>&nbsp; <a href="https://www.youtube.com/watch?v=Ulp1Kimblg0" target="_blank">Swift</a></th>
```
Comment (harikesh409) [inline index.html:213]: Use font awesome swift icon.

File: index.html [hunk 7]
```diff
@@ -207,9 +208,9 @@ <h1 class="text-center">Sharpen Your Skills</h1>
         <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=eTXd89t8ngI&list=PLd3UqWTnYXOmx_J1774ukG_rvrpyWczm0" target="_blank">Java</a></th>
       </tr>
       <tr>
-        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=g-1Cn3ccwXY" target="_blank">STL</a></th>
-        <th><i class="devicon-devicon-plain"></i>&nbsp; <a href="https://clist.by/" target="_blank">Contests</a></th>
-        <th><i class="devicon-redhat-plain"></i>&nbsp; <a href="https://devpost.com/hackathons" target="_blank">Hackathons</a></th>
+        <th><i class="devicon-ruby-plain"></i>&nbsp; <a href="https://www.youtube.com/watch?v=t_ispmWmdjY" target="_blank">Ruby</a></th>
+        <th><i class="devicon-go-plain"></i>&nbsp; <a href="https://www.youtube.com/watch?v=YS4e4q9oBaU" target="_blank">Golang</a></th>
```
Comment (harikesh409) [inline index.html:212]: Keep the G icon or some other font awesome icon which might be related.

File: index.html [hunk 8]
```diff
@@ -23,12 +23,13 @@
...
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.11.0/devicon.min.css">
+  
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.10.1/devicon.min.css"> 
     <link rel="apple-touch-icon" sizes="180x180" href="favicon_package\apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="favicon_package\favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="favicon_package\favicon-16x16.png">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.11.0/devicon.min.css">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.11.0/devicon.min.css">
+
```
Comment (harikesh409) [inline index.html:32]: remove the blank line.

File: index.html [hunk 9]
```diff
@@ -23,12 +23,13 @@
     <link rel="stylesheet" href="css/style.css">
 
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.11.0/devicon.min.css">
```
Comment (harikesh409) [inline index.html:26]: As suggested change to fontawesome icons and remove this additional icon pack.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by codebuzzer01** [comment]
@harikesh409  review it!

Screenshot (225)

File: index.html [hunk 10]
```diff
@@ -166,229 +169,169 @@
...
+   <section id="about" class="about">
+
+    <div class="container">
+        <h1 class="text-center">About Us</h1>
+
+        <div class="row">
+            <div class="col-lg-7">
+                <p class="about-content">
+                    “Algorithms are not arbiters of objective truth and fairness simply because they're math.” - <i><b>Zoe Quinn</b></i>
+
```
Comment (harikesh409) [inline index.html:182]: no need of empty line gap.

File: index.html [hunk 11]
```diff
@@ -166,229 +169,169 @@
...
-                            target="_blank">Ruby</a></th>
-                    <th><i class="fab fa-gofore"></i>&nbsp; <a href="https://www.youtube.com/watch?v=YS4e4q9oBaU"
-                            target="_blank">Golang</a></th>
-                    <th><i class="fab fa-swift"></i>&nbsp; <a href="https://www.youtube.com/watch?v=Ulp1Kimblg0"
-                            target="_blank">Swift</a></th>
-                </tr>
-            </table>
+   <!-- about starts -->
+   <section id="about" class="about">
+
```
Comment (harikesh409) [inline index.html:174]: no need of empty line gap.

File: index.html [hunk 12]
```diff
@@ -166,229 +169,169 @@
...
+    <div class="container">
+        <h1 class="text-center">About Us</h1>
+
```
Comment (harikesh409) [inline index.html:177]: no need of empty line gap.

File: index.html [hunk 13]
```diff
@@ -166,229 +169,169 @@
...
+            <div class="col-lg-7">
+                <p class="about-content">
+                    “Algorithms are not arbiters of objective truth and fairness simply because they're math.” - <i><b>Zoe Quinn</b></i>
+            
+                    <br> <br>
+                  <p class="text-justify">
+                    We are an organization of people who focuses on Data Structure and Algorithms. It is our goal to provide access to well written, well-explained programs and knowledge that improve people's lives. We want to enable people to adapt and
+                    change code according
+                    to their own ideas through an Open Access approach. We bring amazing people together to code algorithms and share our work for the benefit of all.
+
```
Comment (harikesh409) [inline index.html:188]: no need of empty line gap.

File: index.html [hunk 14]
```diff
@@ -166,229 +169,169 @@
...
+                    to their own ideas through an Open Access approach. We bring amazing people together to code algorithms and share our work for the benefit of all.
+                
+            </div>
+            <div class="col-lg-5 d-flex align-items-center justify-content-center">
+                <img src="images/about_us_background.jpg" alt="" class="img-fluid">
+            </div>
         </div>
+    </div>
+  </section>
+
```
Comment (harikesh409) [inline index.html:196]: no need of empty line gap.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Review by harikesh409 — APPROVED** [review]

**Comment by harikesh409** [comment]
@codebuzzer01 please resolve the merge conflicts.

**Comment by codebuzzer01** [comment]
@harikesh409  resolved !

**Comment by harikesh409** [comment]
> @harikesh409 resolved !

Still conflicts are there look into it.

**Comment by codebuzzer01** [comment]
Hey, @harikesh409  It should have a label of 'level' . You didn't assign it any.

**Comment by harikesh409** [comment]
> Hey, @harikesh409 It should have a label of 'level' . You didn't assign it any.

Updated the labels.

**Pull request merged**